server/channels/app/platform/web_broadcast_hook_test.go

```diff
@@ -72,7 +72,7 @@ func TestRunBroadcastHooks(t *testing.T) {
 		result := hub.runBroadcastHooks(event, webConn, hookIDs, hookArgs)
 
 		assert.NotSame(t, event, result)
-		assert.NotSame(t, event.GetData(), result.GetData())
+		assert.NotSame(t, model.NewPointer(event.GetData()), model.NewPointer(result.GetData()))
```

Comment:
This is actually "wrong" as explained in https://github.com/mattermost/mattermost/pull/34058, but I'm leaving it to match `master` for now. (It has no impact on correctness.)